yolox/evaluators/coco_evaluator.py

```diff
@@ -254,7 +279,9 @@ def evaluate_prediction(self, data_dict, statistics):
                 cocoEval.summarize()
             info += redirect_string.getvalue()
             if self.per_class_mAP:
-                info += "per class mAP:\n" + per_class_mAP_table(cocoEval)
+                info += "per class mAP:\n" + per_class_mAP_table(cocoEval) + "\n"
+            if self.per_class_AR:
```

Comment:
AttributeError: 'COCOEvaluator' object has no attribute 'per_class_AR'

Reply:
#1065

Reply:
@759401524 fixed now, please pull the latest branch.